pylint/__pkginfo__.py

```diff
@@ -34,7 +34,7 @@
 else:
     version = string_version
 
-install_requires = ["astroid>=2.2.0", "isort >= 4.2.5", "mccabe"]
+install_requires = ["astroid>=2.2.0,<3", "isort>=4.2.5,<5", "mccabe>=0.6,<0.7"]
```

Comment:
mccabe is pinned more strictly (to within the same minor version) since it is a major version 0 package, which semver says may make breaking releases between minor versions.

Reply:
It turns out astroid 2.2.0 was a breaking release over 2.1.0. Pinned to 2.1.0 in #2774.